bokeh/models/renderers.py

```diff
@@ -26,7 +26,14 @@ class Renderer(Model):
     generally useful to instantiate on its own.
 
     """
-class TileRenderer(Renderer):
+
+@abstract
+class DataRenderer(Renderer):
+    """ A base class for data renderer types (e.g. GlyphRenderer). ``DataRenderer`` is not
+    generally useful to instantiate on its own.
```

Comment:
No. Abstract classes can not be instantiated, but I see where this comes from. Please change `Renderer`'s docstring as well. "An abstract base class for ..." should suffice. Also, `GlyphRenderer` should be enclosed with double backticks.